Luke_Smith_Portfolio.css

```diff
@@ -0,0 +1,126 @@
+body {aaa
+  margin: auto;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.topnav {
+  overflow: auto;
+  background-color: #ffffff;
+}
+
+.topnav a {
+  float: left;
+  color: #000000;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+  width: auto;
+}
+
+.topnav a:hover {
+  background-color: #dddddd;
+  color: #000000;
+}
+
+body {
+  background: #555555;
+}
+
+/*About Me Section*/
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  color: #000000;
+  max-width: 500px;
+  margin: 0 auto;
```

Comment:
Move this to the body {} styling at the top of the page. All three of your body{} styling rules are applying to the same elements, so you only need one styling rule for the body tag.